Function/FunctionsSchema.xsd

```diff
@@ -57,6 +57,7 @@
                         </xs:sequence>
                     </xs:complexType>
                 </xs:element>
+                <xs:element name="OverrideTimeoutVF" type="xs:string" minOccurs="0"/>
```

Comment:
can the type be updated so that the possible values are only true or false? See e.g. EnumTrueFalseLowerCase in Automation XSD: https://github.com/SkylineCommunications/Skyline.DataMiner.XmlSchemas/blob/main/Automation/automation.xsd

Reply:
updated